readme.md

```diff
@@ -48,35 +55,45 @@ Life Cycle of a request/response in Rails:
 
 4. Once the request hits the controller, its then going to query the database through Active Record(the model) for the information specified in the controller.
 
-5. Once the controller has the information from the model that it needs it sends it to the view
+5. Once the controller has the information from the model that it needs, it sends it to the view
+
+6. The view takes the objects from the controller, and sends a response to the user.
 
-6. The view takes the objects from the controller and sends a response to the user.
+Let's dive into an example, and see MVC in action by writing some Rails code.
 
-Let's dive into an example, and see MVC in action by writing some rails code.
+## Set-Up (5 / 30)
 
-## Set-Up (5/30)
-This lesson will build directly off of the work you did with Models and Migrations.
+This lesson will build off of the completed work you will do in the Models and Migrations lesson, so just know we will not be going in depth into the database migrations or have to write any new Model code.
 
-If you would prefer to work with a clean starter rather than building on your existing copy:
+To get started, clone down the following repo:
 
 ```bash
-git clone git@github.com:ga-dc/tunr_rails_models_and_migrations.git
+git clone git@github.com:ga-wdi-exercises/tunr_rails_models_and_migrations.git
 cd tunr_rails_models_and_migrations
 git checkout views-controllers-starter
 git checkout -b inclass
 ```
-Make sure to remember to run:
+After you start working on a new branch, from the terminal run:
 ```
 bundle install
 rake db:create
-# ^Only if you do not have a tunr_db database
-rake db:migrate
+rake db:seed
+```
+
+> Postgres troubles? Is your blue elephant running?
+
+Here, we are just installing our app's dependencies, and running the setup for our app's database locally.
+
+To test that it works, try starting the server:
+
+```
+rails s
```

Comment:
"What does it mean to 'start a server'?"
"It means you've told your computer to start listening for requests being made to a specific URL. In this case, the URL is `localhost:3000`. The `3000` is a "port". Your computer can listen for requests coming from thousands of directions, as if it was a secretary holding thousands of office phones. 3000 is one of them."